src/signals/incident-management/containers/IncidentDetail/components/StatusForm/components/EmailPreview/EmailPreview.tsx

```diff
@@ -35,27 +35,40 @@ interface EmailPreviewProps {
   onUpdate: () => void
 }
 
-const fontStyling = '<style>*{font-family:Avenir Next;}</style>'
+const styling = '<style>*{font-family:"Avenir Next";}</style>'
+const fontSrc =
+  '<link rel="stylesheet" href="https://static.amsterdam.nl/fonts/fonts.css"/>'
 
 const EmailPreview: FC<EmailPreviewProps> = ({
   emailBody,
   onUpdate,
   onClose,
-}) => (
-  <ModalContainer>
-    <ModalHeader title="Controleer bericht aan melder" onClose={onClose} />
-    <StyledIframe
-      data-testid="emailBodyIframe"
-      srcDoc={emailBody.concat(fontStyling)}
-      height="500"
-    />
-    <StyledFormFooter
-      cancelBtnLabel="Wijzig"
-      onCancel={onClose}
-      submitBtnLabel="Verstuur"
-      onSubmitForm={onUpdate}
-    />
-  </ModalContainer>
-)
+}) => {
+  const htmlParts = emailBody.split(/(<head>)/)
+  const styledHtml = [
+    ...htmlParts[0],
+    ...htmlParts[1],
+    ...fontSrc,
+    ...styling,
+    ...htmlParts[2],
```

Comment:
This is a very fragile construct; if anything in the HTML structure changes, the indexes might not match the expected output anymore.
Might be better to do something like this:
```javascript
const body = emailBody.replace('</head>', `${fontSrc}${styling}</head>`)
```

Reply:
you proposal is more concise, but not less fagile. it equally depends on the presence of the <head> tag.

Reply:
The `<head>` tag can potentially have attributes. A closing tag can never.